deploy/streisand-new-cloud-server.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+#
+# Run a noninteractive Streisand installation that creates a new cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# streisand-new-cloud-server \
+#   --provider [amazon|azure|digitalocean|google|linode|rackspace] \
+#   --site-config path/to/digitalocean-site.yml
+#
+
+set -o errexit
+set -o nounset
+
+DIR="$( cd "$( dirname "$0" )" && pwd)"
+PROJECT_DIR="${DIR}/.."
+
+VALID_PROVIDERS="amazon|azure|digitalocean|google|linode|rackspace"
+export DEFAULT_SITE_VARS="${PROJECT_DIR}/global_vars/default-site.yml"
+export GLOBAL_VARS="${PROJECT_DIR}/global_vars/vars.yml"
+
+# Include the check_ansible function from ansible_check.sh.
+# shellcheck source=util/source_validate_and_deploy.sh
+source "${PROJECT_DIR}/util/ansible_check.sh"
+check_ansible
+
+# --------------------------------------------------------------------------
+# Reading options.
+# --------------------------------------------------------------------------
+
+function usage () {
+  cat <<EOF
+Usage:
+$0 \\
+  --provider [${VALID_PROVIDERS}] \\
+  --site-config path/to/site.yml
+EOF
+}
+
+PROVIDER=""
+SITE_VARS=""
+
+while [[ ${#} -gt 0 ]]; do
+  case "${1}" in
+    # Required.
+    --provider)      PROVIDER="${2}"; shift;;
+    --site-config)   SITE_VARS="${2}"; shift;;
+
+    # Utility.
+    -h|--help)       usage; exit 0;;
+    --)              break;;
+    -*)              echo "Unrecognized option ${1}"; usage; exit 1;;
+  esac
+
+  shift
+done
+
+# --------------------------------------------------------------------------
+# Fail if required options are not set.
+# --------------------------------------------------------------------------
+
+if [ -z "${PROVIDER}" ] || [ -z "${SITE_VARS}" ]; then
+  usage
+  exit 1
+fi
+
+# --------------------------------------------------------------------------
+# Fail for other reasons.
+# --------------------------------------------------------------------------
+
+# Make sure the alleged configuration file exists.
+if [ ! -f "${SITE_VARS}" ]; then
```

Comment:
Can you match the `${SITE_VARS}` handling here with how it's done in `deploy/streisand-local.sh`? I don't think it will default the same way.